src/main/java/frc/robot/RobotContainer.java

```diff
@@ -147,22 +149,48 @@ private void sysIdToShuffleboard() {
 
   private void configureBindings() {
     // shooter.setDefaultCommand(new InstantCommand(() -> shooter.disable(), shooter));
-    controller.rightTrigger().onTrue(new ScorePiece(RobotConfig.intake, RobotConfig.shooter));
 
-    controller.a().onTrue(new PrepareForIntake(RobotConfig.arm, RobotConfig.intake));
+    // Activate shooter to score piece in speaker or amp
+    controller
+        .rightTrigger()
+        .onTrue(new ScorePiece(RobotConfig.intake, RobotConfig.shooter).withTimeout(1));
 
-    controller.b().onTrue(new PrepareForScore(RobotConfig.arm, RobotConfig.shooter, () -> ampMode));
+    // Move arm to some "safe" angle to protect intake while driving.
+    controller
+        .leftTrigger()
+        .onTrue(
+            new ArmToPositionTP(
+                () -> RobotConfig.ArmConstants.stowIntakeAngleInDegrees, RobotConfig.arm));
+
+    // Try to spit piece out of intake in case it gets stuck
+    controller.leftBumper().onTrue(new EjectPiece(RobotConfig.intake, RobotConfig.arm));
 
+    controller.a().onTrue(new PrepareForIntake(RobotConfig.arm, RobotConfig.intake));
+    controller.b().onTrue(new PrepareForScore(RobotConfig.arm, RobotConfig.shooter, () -> ampMode));
     controller
         .y()
-        .onTrue(
-            new InstantCommand(
-                () -> {
-                  ampMode = !ampMode;
-                  if (ampModeEntry != null) {
-                    ampModeEntry.setBoolean(ampMode);
-                  }
-                }));
+        .onTrue(new PrepareForScore(RobotConfig.arm, RobotConfig.shooter, () -> !ampMode));
```

Comment:
Please update https://docs.google.com/document/d/1A22hMpvmXuujwkwF299oZQ94aNbzZQalUD1ifmwRZtk with any changes to the controls before pushing to baseline.

Reply:
Done, updated Basic Dev/Debug Controls section.